index.js

```diff
@@ -21,15 +21,24 @@ module.exports = {
     restParams: true,
     unicodeCodePointEscapes: false,
     experimentalObjectRestSpread: true,
-    jsx: true
+    jsx: true,
   },
   rules: {
     'accessor-pairs': 2,
     'arrow-body-style': [0, 'as-needed'],
     // TODO: Consider if we want to enable this rule.
     'arrow-parens': 0,
     'class-methods-use-this': 0,
-    'comma-dangle': [0, 'always-multiline'],
+    'comma-dangle': [
+      0,
+      {
+        arrays: 'always-multiline',
+        objects: 'always-multiline',
+        imports: 'always-multiline',
+        exports: 'always-multiline',
+        functions: 'ignore',
```

Comment:
without `functions: ignore`, `prettier-eslint` will assume we want trailing commas _everywhere_, including in function arguments. while this gets compiled away in babel anyway, its a little strange to me, and in any case, this just retains this functionality